lib/finsta/posts/post.ex

```diff
@@ -2,6 +2,13 @@ defmodule Finsta.Posts.Post do
   use Ecto.Schema
   import Ecto.Changeset
 
+  @required_fields ~w(
+    caption
+    user_id
+  )a
+
+  @allowed_fields ~w()a ++ @required_fields
+
```

Comment:
Even though I do like this, I'd try to introduce the less amount of changes to keep it simple. Let me know what you think